projectile.el

```diff
@@ -1146,7 +1146,7 @@ Controlled by `projectile-require-project-root'."
     (cond
      ((eq projectile-require-project-root 'prompt) (projectile-completing-read
                                                     "Switch to project: " projectile-known-projects))
-     (projectile-require-project-root (error "Projectile can't find a project definition in %s" dir))
+     (projectile-require-project-root (error "Projectile can’t find a project definition in %s" default-directory))
```

Comment:
@bbatsov I managed to resolve [the issue](https://github.com/bbatsov/projectile/pull/1612#discussion_r535689077) by replacing `'` with `’` here. Note that the tests only failed for a few versions of Emacs (the latest ones). This trick should make it compatible with all versions of Emacs and still allow us to write an assertion to test the error message. Let me know if you're OK with this.